internal/msgstream/mq_msgstream_test.go

```diff
@@ -29,21 +29,24 @@ import (
 	"time"
 	"unsafe"
 
+	pulsar2 "github.com/milvus-io/milvus/internal/msgstream/mqclient/pulsar"
```

Comment:
we'd better not name the folder as pulsar otherwise the github.com/milvus-io/milvus/internal/msgstream/mqclient/pulsar and github.com/apache/pulsar-client-go/pulsar get conflicts